pkg/kubelet/kuberuntime/security_context_others.go

```diff
@@ -47,6 +48,8 @@ func verifyRunAsNonRoot(pod *v1.Pod, container *v1.Container, uid *int64, userna
 		return fmt.Errorf("container has runAsNonRoot and image will run as root (pod: %q, container: %s)", format.Pod(pod), container.Name)
 	case uid == nil && len(username) > 0:
 		return fmt.Errorf("container has runAsNonRoot and image has non-numeric user (%s), cannot verify user is non-root (pod: %q, container: %s)", username, format.Pod(pod), container.Name)
+	case uid != nil && validation.IsValidUserID(*uid) != nil:
+		return fmt.Errorf("container has runAsNonRoot and image has an invalid user id (%d): %s (pod: %q, container: %s)", *uid, validation.IsValidUserID(*uid), format.Pod(pod), container.Name)
```

Comment:
would it be helpful to say what the valid id range is?

Reply:
`validataion.IsValidUserID` returns slice of strings. Should we loop over them here the same way it's done in other places where it's called?

Reply:
Can we avoid calling `validation.IsValidUserID(*uid)` twice?